lib/manager/components/version/FeedVersionDetails.js

```diff
@@ -111,6 +111,28 @@ export default class FeedVersionDetails extends Component<Props> {
     const mergeButtonLabel = isMergedServicePeriods
       ? 'Cannot re-merge feed'
       : 'Merge with version'
+
+    const hasMtcExtension = isExtensionEnabled('mtc')
+    let publishWarningMessage
+    if (hasMtcExtension) {
+      if (hasBlockingIssue || hasGtfsPlusBlockingIssue) {
+        publishWarningMessage = (
```

Comment:
Do you think making this an internationalized message help clean up the `{' '}`'s here?

Reply:
The feed merge handling has not really been included in the localization stuff, so I won't do it in this PR.